stackdriver/non_rack_debugger.rb

```diff
@@ -0,0 +1,13 @@
+# [START explicit_debugger_norack]
```

Comment:
`non_rack_debugger.rb` --> `ruby_debugger.rb`

Let's not "non-rack". It's simply pure Ruby 😄